owncloudApp/src/main/res/values/strings.xml

```diff
@@ -832,5 +832,7 @@
     <string name="audio_preview_label">Audio preview</string>
     <string name="details_label">Details</string>
     <string name="text_preview_label">Text preview</string>
+    <string name="release_notes_title_enhanced_bottom_nav_bar">Added text labels on bottom nav bar</string>
```

Comment:
This is going to be read by all users, so let's not use technical language. Bottom bar is OK 👍 
```suggestion
    <string name="release_notes_title_enhanced_bottom_nav_bar">Added text labels on bottom bar</string>
```